src/components/app/App.js

```diff
@@ -1,8 +1,55 @@
 import React from 'react';
-import { ShowContext, ShowContextProvider } from '../../contexts/ShowContext';
+import { ShowContextProvider } from '../../contexts/ShowContext';
+import { Route, Switch } from 'react-router-dom';
+import Navbar from './components/navbar/Navbar';
+import Container from './components/container/Container';
+import Footer from './components/footer/Footer';
 
 function App() {
-  return <ShowContextProvider></ShowContextProvider>;
+  return (
+    <ShowContextProvider>
+      <Switch>
+        <Route
```

Comment:
What path is the homepage? Typically the homepage is just "/". Does that route exist? If not, do you want people to have to type out a URL with a longer path to visit your page? Do you want them to get an error for trying to go to the base URL? (Consider how google.com, twitter.com, facebook.com do not make you add anything else to their URL path to get to their site.)